lib/stm32/f1/rcc.c

```diff
@@ -637,10 +637,10 @@ void rcc_clock_setup_in_hsi_out_64mhz(void)
 	rcc_set_ppre2(RCC_CFGR_PPRE2_HCLK_NODIV);   /* Set. 64MHz Max. 72MHz */
 
 	/*
-	 * Sysclk is running with 64MHz -> 2 waitstates.
-	 * 0WS from 0-24MHz
-	 * 1WS from 24-48MHz
-	 * 2WS from 48-72MHz
+	 * SYSCLK will run at 64MHz -> 2 waitstates.
+	 * 0WS if 0MHz < SYSCLK <= 24MHz
+	 * 1WS if 24MHz < SYSCLK <= 48MHz
+	 * 2WS if 48MHz < SYSCLK <= 72MHz
 	 */
```

Comment:
This is a fixed function, if you want to touch these comments, just delete them IMO